examples/qasm.ipynb

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
<p>There is a warning message here, explain that it is expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>